src/db/forum-questions.ts

```diff
@@ -11,6 +11,7 @@ export const forumQuestions = pgTable('forum_questions', {
     .notNull(),
   questionTitle: varchar('question_title', { length: 256 }).notNull(),
   questionSubTitle: varchar('question_sub_title', { length: 256 }),
+  voteModel: varchar('vote_model', { length: 256 }).notNull(),
```

Comment:
making this not null means that there has to be a default as it did not exist before.

Reply:
added COCM as default.